.env_sample

```diff
@@ -0,0 +1,4 @@
+RDS_HOST="A HOST"
+RDS_NAME="A DATABSE"
+RDS_username="A USERNAME"
```

Comment:
These will actually cause a failure when copied and used to run the app since your config uses `RDS_USERNAME` and `RDS_PASSWORD`. The case does matter and other devs would have unkind words for you.